packages/server/test/course.integration.ts

```diff
@@ -719,7 +755,7 @@ describe('Course Integration', () => {
         })
         .expect(200);
 
-      const checkinTimes = ((data.body as unknown) as TACheckinTimesResponse)
+      const checkinTimes = (data.body as unknown as TACheckinTimesResponse)
```

Comment:
Is there a way to do that as `as (unknown || TACheckinTimesResponse)`? Also, why we are modifying that?

Reply:
I think the auto-formatting did that